docs/platforms/apple/guides/ios/session-replay/index.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: Set Up Beta Session Replay for Mobile
+sidebar_order: 5500
+notSupported:
+description: "Learn how to enable the Beta of Mobile Session Replay in your app."
+---
+
+<Note>
+
+Mobile support for Session Replay is in Beta. Features available in Beta are still work-in-progress and may have bugs. We recognize the irony.
+
+If you have any questions, feedback or would like to report a bug, please open a [GitHub issue](https://github.com/getsentry/sentry-cocoa/issues/new?assignees=&labels=Platform%3A+Cocoa%2CType%3A+Bug&projects=&template=bug.yml) with a link to a relevant replay in Sentry if possible.
+
+</Note>
+
+[Session Replay](/product/explore/session-replay/) helps you get to the root cause of an error or latency issue faster by providing you with a reproduction of what was happening in the user's device before, during, and after the issue. You can rewind and replay your application's state and see key user interactions, like taps, swipes, network requests, and console entries, in a single UI.
+
+By default, our Session Replay SDK masks all text content, images, and user input, giving you heightened confidence that no sensitive data will leave the device. To learn more, see <PlatformLink to="/product/session-replay/">Session Replay Product docs</PlatformLink>.
+
+## Pre-requisites
```

Comment:
```suggestion
## Prerequisites
```

Reply:
Requires a re-write docs-wide since this is the form used everywhere